### PR TITLE
feat(ga): add a datasource to get list of availability zones

### PR DESCRIPTION
**Pull request by ruwenqiang123**
**What this PR does / why we need it**:

Add a datasource to get list of availability zones.

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
fixes #xxx

**Special notes for your reviewer**:

**Release note**:

```release-note
1.support a new datasource to get the list of the availability zones.
2.support related document and acceptance test.
```

## PR Checklist

* [x] Tests added/passed.
* [x] Documentation updated.
* [x] Schema updated.

## Acceptance Steps Performed

```
$ make testacc TEST="./huaweicloud/services/acceptance/ga" TESTARGS="-run TestAccDataSourceGaAvailabilityZones_basic"
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./huaweicloud/services/acceptance/ga -v -run TestAccDataSourceGaAvailabilityZones_basic -timeout 360m -parallel 4
=== RUN   TestAccDataSourceGaAvailabilityZones_basic
=== PAUSE TestAccDataSourceGaAvailabilityZones_basic
=== CONT  TestAccDataSourceGaAvailabilityZones_basic
--- PASS: TestAccDataSourceGaAvailabilityZones_basic (23.44s)
PASS
ok      github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance/ga        23.501s
```

### CODE REVIEW

File: docs/data-sources/ga_availability_zones.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  The [regions](#regions_struct) structure is documented below.
+
+<a name="regions_struct"></a>
+The `regions` block supports:
+
+* `region_id` - The region ID.
+
+* `area` - The acceleration area to which the region belongs.
+
+* `endpoint_types` - The endpoint type supported by the region.
```
Comment (deer-hang) [inline docs/data-sources/ga_availability_zones.md]:
```suggestion
* `endpoint_types` - The endpoint types supported by the region.
```

Comment (ruwenqiang123) [reply docs/data-sources/ga_availability_zones.md]: done

**Pull request merged**